.github/workflows/playwright.yml

```diff
@@ -1,11 +1,13 @@
-name: Playwright Tests
+name: Fv-Apps
```

Comment:
Rather than modifying this action, can you please add a new one? We do still want the ability to run the playwright tests

Reply:
I just renamed the pipeline name as it contains more than just tests now. There is a job/step in the pipeline which can be enabled or disabled whenever needed.